wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/ui/episode/EpisodeScreen.kt

```diff
@@ -283,6 +294,46 @@ private fun EpisodeListChip(episodeScreenItem: EpisodeScreenItem) {
     )
 }
 
+@Composable
+private fun EpisodeErrorDetails(
+    errorData: EpisodeViewModel.State.Loaded.ErrorData,
+) {
+    val padding = 8.dp
+    Row(
+        modifier = Modifier
+            .padding(start = padding, end = padding, bottom = padding)
+            .background(
+                color = MaterialTheme.colors.secondary,
+                shape = RoundedCornerShape(10.dp)
+            ),
+    ) {
+        Box(modifier = Modifier.padding(start = padding, top = 10.dp)) {
+            Icon(
+                painter = painterResource(errorData.errorIconRes),
+                contentDescription = null,
+                modifier = Modifier.size(16.dp)
+            )
+        }
+
+        Column {
+            Text(
+                text = stringResource(errorData.errorTitleRes),
+                color = MaterialTheme.colors.onSecondary,
+                style = MaterialTheme.typography.caption2,
+                modifier = Modifier.padding(padding)
+            )
+            errorData.errorDescription?.let {
+                Text(
+                    text = it,
+                    color = MaterialTheme.colors.onSecondary,
+                    style = MaterialTheme.typography.caption3,
+                    modifier = Modifier.padding(start = padding, end = padding, bottom = padding)
+                )
+            }
```

Comment:
It doesn't seem like the title adds much info. I feel like we might be better off just displaying the description without the title and only using the title if the description is null.

Reply:
Changed in 53854a0

![Screenshot_20230601_152750](https://github.com/Automattic/pocket-casts-android/assets/1405144/c6940510-96ea-47f3-b91a-852fe4436655)